src/services/webhook/webhookService.ts

```
@@ -76,23 +76,23 @@ export async function enqueueWebhook(
   return webhook.id;
 }
 
-export async function deliverWebhook(webhookId: string): Promise<boolean> {
+export async function deliverWebhook(webhookId: string): Promise<{ success: boolean; terminal: boolean }> {
```

ESLint

[error] Replace `webhookId:·string` with `⏎··webhookId:·string,⏎`

(prettier/prettier)